includes/admin_meeting.php

```diff
@@ -235,6 +238,12 @@ function () {
             <input value="<?php tsml_echo($location, 'approximate') ?>" type="hidden" name="approximate" id="approximate">
             <input value="<?php tsml_echo($location, 'latitude') ?>" type="hidden" name="latitude" id="latitude">
             <input value="<?php tsml_echo($location, 'latitude') ?>" type="hidden" name="longitude" id="longitude">
+            <small class="error_message" data-message="1">
+                <?php _e('Error: In person meetings must have a specific address.', '12-step-meeting-list') ?>
+            </small>
+            <small class="warning_message" data-message="2">
+                <?php _e('Warning: Unable to process this address for exact location.', '12-step-meeting-list') ?>
+            </small>
```

Comment:
I added this error state to capture when we get an error from the geocoding call. I figured it was worth displaying, but you may know reasons why this isn't useful - maybe frequent false positives?

Reply:
no this is great - we can now cross this requirement off our [refactor geocoding](https://github.com/code4recovery/12-step-meeting-list/issues/1002) ticket